core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -1,7 +1,7 @@
 <!--
 The MIT License
 
-Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
+Copyright (c) 2004-2015, Sun Microsystems, Inc., Kohsuke Kawaguchi,
```

Comment:
I highly doubt that this is copyright in this range by Sun Microsystems.
If you feel like modifying this then add a new line with some new dates and the new copyright holder.